scripts/install_latest_build2.sh

```diff
@@ -9,10 +9,23 @@ fi
 
 install_dir=$1
 toolchain_file="toolchain.sha256"
-wget --no-verbose --no-clobber https://download.build2.org/toolchain.sha256 -O "${toolchain_file}"
+if [ "$1" = "--version" ]; then
+    wget -q --no-verbose --no-clobber https://download.build2.org/toolchain.sha256 -O "${toolchain_file}"
+else
+    wget --no-verbose --no-clobber https://download.build2.org/toolchain.sha256 -O "${toolchain_file}"
+fi
```

Comment:
What is the diff here? I only see a `-q` flag.

Reply:
The full change is to give the script the ability to print which version would it install in case someone called it with the `--version` argument, without actually installing it. I think it is a better approach to exclude the output of `wget` and only print out the version number in this case. Since the `install_latest_build2.sh` script won't change even a new build2 version is out, I needed a mechanism to cache build2 in a cheap manner, so I use the version output to keep track of it.